src/svc/telemetry/prometheus.rs

```diff
@@ -50,33 +50,85 @@ impl LabeledMetric {
         self.metric_name.replace('.', "_")
     }
 
+    /// Create a type line, typically:
+    ///
     /// # TYPE protocol_https gauge
     fn type_line(&self) -> String {
         let printable_metric_name = self.printable_name();
         format!("# TYPE {} {}", printable_metric_name, self.metric_type)
     }
 
+    /// Format labels in a comma-separated list:
+    ///
+    /// ```plain
+    /// "label"="value","other"="value"
+    /// ```
+    fn formatted_labels(&self) -> String {
+        self.labels
+            .iter()
+            .map(|(name, value)| format!("{}=\"{}\"", name, value))
```

Comment:
Use `Iterator`'s `chain` instead of `format!`

Reply:
for two items it is not yet worth it